### PR TITLE
Can only see personnel if user is pdl Closes #292

### LINKED ISSUE #292: FE > Only display My Personnel panel if the user is a PDL
On the check-ins page, this panel should only show if the current user is a PDL.

### PR DESCRIPTION
**Pull request by jahanner**

### CODE REVIEW

File: web-ui/src/pages/CheckinsPage.jsx [hunk 1]
```diff
@@ -13,8 +13,10 @@ const CheckinsPage = () => {
   const [show, setShow] = useState(false);
   const { state } = useContext(AppContext);
   const { checkins, userProfile } = state;
+  console.log({ userProfile });
   const [index, setIndex] = useState(0);
   const checkin = checkins[index];
+  const canSeePersonnel = userProfile && userProfile.role.includes("PDL");
```
Comment (bhatiam12) [inline web-ui/src/pages/CheckinsPage.jsx:19]: can ADMIN also view it ? do we need another check for ADMIN role ?

Comment (jahanner) [reply web-ui/src/pages/CheckinsPage.jsx:19]: The story said only if the user is a PDL, but maybe we do want admin to see it too? @mkimberlin What do you think?

Comment (mkimberlin) [reply web-ui/src/pages/CheckinsPage.jsx:19]: Only PDLs. If an ADMIN is also a PDL, yes... If not, then it doesn't make sense for them to see it.

File: web-ui/src/pages/CheckinsPage.jsx [hunk 2]
```diff
@@ -13,8 +13,10 @@ const CheckinsPage = () => {
   const [show, setShow] = useState(false);
   const { state } = useContext(AppContext);
   const { checkins, userProfile } = state;
+  console.log({ userProfile });
```
Comment (mkimberlin) [inline web-ui/src/pages/CheckinsPage.jsx:16]: I'd remove this line.

**Review by mkimberlin — APPROVED** [review]
Kill that `console.log` and then you are good.

File: web-ui/src/pages/CheckinsPage.jsx (hunk 2 above)
Comment (jahanner) [reply web-ui/src/pages/CheckinsPage.jsx:16]: Mille gratzie

**Pull request merged**